GVFS/GVFS.UnitTests/Platform.Mac/MacFileSystemVirtualizerTests.cs

```diff
@@ -108,7 +107,7 @@ public void UpdatePlaceholderIfNeeded()
                 mockVirtualization.UpdatePlaceholderIfNeededFailureCause = UpdateFailureCause.NoFailure;
                 virtualizer
                     .UpdatePlaceholderIfNeeded(
-                        "test.txt",
+                        filePath,

```

Comment:
This changed from `"test.txt"` to `"test/test.txt"`  Was that intended?

Reply:
Yep this was to match the path we're using above when adding the test file to `gitIndexProjection.MockFileTypesAndModes`.

Reply:
Actually, I need to dig more into this and see how the mock paths are getting set up.

Thanks for pointing this out!